avalon/tools/cbsceneinventory/proxy.py

```diff
@@ -63,6 +63,13 @@ def filterAcceptsRow(self, source_row, source_parent):
         column = self.filterKeyColumn()
         index = model.index(source_row, column, source_parent)
 
+        # Filters below are per version group (top-level entries) so if
```

Comment:
Hm, I can't make sense out what this comment means. Do you think you can reformulate it?